README.md

```diff
@@ -55,6 +55,8 @@ If you do not supply a `default` configuration, exceptions will be raised. If yo
 
 All `nil` values will fallback to their default values. `SecureHeaders::OPT_OUT` will disable the header entirely.
 
+**Word of caution:**  The following is not a default configuration per se. It serves as a reference to all the available options. You should read more about these headers and determine what is appropriate for your requirements. 
```

Comment:
> It serves as a reference to all the available options.

This isn't quite true since not all of the directives have all available options. For example, HSTS:

```rb
# Add "; preload" and submit the site to hstspreload.org for best protection.
config.hsts = "max-age=#{20.years.to_i}; includeSubdomains"
```

Perhaps wording along the lines of "It serves as a sample implementation of the configuration"? Alternatively, we could go through and add all the possible values however I think this opens a bigger can of worms for those who don't read this warning and just copy-pasta into their app and all of a sudden, their site is bricked because of incorrectly configured HSTS or HPKP.

Reply:
We could also just treat those dangerous directives differently and allow them to not be the full available options so people cannot shoot themselves in the foot without knowing it.